src/backend/gamelauncher.h

```diff
@@ -33,30 +35,37 @@ namespace launcher {
 
 
     // Validates game is an exe file and fixes escape characters in the path.
-    // Input:   A string address representing game's path.
+    // Inputs:   A string address representing game's path (path) and a string
+    //           to represent the file type (file_type).
     void EnhancePath(string& path, string file_type) ;
 
 
-    // used for adding a game
-    // Inputs:
-    // game_name: name of the game
-    // game_path: path of the game
-    // image_path: path of the image
-
+    // Adds a game to the database where it is then stored.
+    // Inputs: The name of the game (game_name), the path of the game (game_path),
+    //         and the path of the image (image_path).
+    // Output: A string to represent the status of the add.
     string add(string game_name, string game_path, string image_path);
 
-    // used for launching the game
+
+    // Plays a game stored in the database with the given id value.
+    // Input:  An integer value that represents the desired id of the game to play.
+    // Output: A boolean value that returns true if play was successful, otherwise false.
     BOOL play(int id);
 
-    // used for deleting a game
-    string del(int id);
 
-    // used for getting updated list of dbObjects
+    // Produces an updated list of dbObjects by displaying what is in the database.
```

Comment:
Leave implementation details for developers, no need to write them in the header doc.